packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

```diff
@@ -1461,22 +1681,38 @@ Content-Type: application/json;charset=UTF-8
 
 #### Rule properties
 
-| Property     | Description                                                                                  | Type                                    | Required for create   | Required for update |
-| :----------- | :------------------------------------------------------------------------------------------- | :-------------------------------------- | :-------------------- | :------------------ |
-| actions      | An object that contains the `tokens` array, which shows lifetime durations for the tokens    | Object                                  | System generated      | System generated    |
-| conditions   | Specifies the people, Groups, grant types, and Scopes that the rule is applied to             | [Condition object](#condition-object)   | False                 | False               |
-| id           | ID of the rule                                                                               | String                                  | False                 | True                |
-| name         | Name of the rule                                                                             | String                                  | True                  | True                |
-| status       | Specifies whether requests have access to this Claim. Valid values: `ACTIVE` or `INACTIVE`   | Enum                                    | True                  | True                |
-| system       | Specifies whether the rule was created by Okta                                        | Boolean                                 | True                  | True                |
+| Property      | Description                                                        | Data Type                                      | Required for Create  | Required for update    |
+| :------------ | :----------------------------------------------------------------- | :--------------------------------------------- | :------------------- | :--------------------- |
+| id            | Identifier of the rule                                             | String                                         | Assigned             | Yes                    |
+| type          | Rule type. Valid values: `RESOURCE_ACCESS`                         | String (Enum)                                  | No                   | No                     |
+| name          | Name of the rule                                                   | String                                         | Yes                  | Yes                    |
```

Comment:
I feel it is better to have `Required for create/update` to have True/False values instead of Yes/No, to be consistent with the rest of the doc. @susanharper-okta Are we changing to Yes/No going forward?

Reply:
Do you set the value in the create/update as "true" and "false" or "yes" and "no"? It should be whatever the actual VALUE is that is set in the request.

Reply:
These are String values, not boolean. For example, is the field `name` required for create/update? This should be True instead of Yes, to be consistent with the rest of the doc.

Reply:
gotcha. yes, please use True and False

Reply:
Will update inline with this discussion